src/core-runtime.js

```diff
@@ -30,6 +30,14 @@ Globalize.locale = function( locale ) {
 	return this._locale;
 };
 
+Globalize._loadRuntimeFn = function( key ) {
+	var fn = Globalize[ key ];
+	if ( typeof fn === "function" ) {
+		fn.runtimeKey = key;
+	}
+	return fn;
+};
```

Comment:
We cannot force, indeed, but there are ways to bring this information to the projects. One is documentation and release notes. Another is to use semver and make the existing globalize-compiler to require a peer globalize that goes up until 1.2.3; Only new globalize-compiler will support a peer globalize 1.3.0.